core/include/scattered_array.h

```diff
@@ -24,7 +24,7 @@
 		__section(section_name __SECTION_FLAGS_RODATA)
 
 #define __SCT_ARRAY_DEF_PG_ITEM3(element_type, element_name, section_name) \
-	static const element_type element_name __used \
+	static const element_type element_name __used __aligned(1) \
```

Comment:
need #if defined(X86_64) here

Reply:
Done, add #if defined(X86_64)

Reply:
Why doesn't Arm need this aligned(1)? GCC doesn't do padding to 32 bytes on Arm?